src/plugins/index_pattern_field_editor/public/open_editor.tsx

```diff
@@ -91,6 +91,13 @@ export const getFieldEditorOpener = ({
     const fieldTypeToProcess: InternalFieldType =
       isNewRuntimeField || isExistingRuntimeField ? 'runtime' : 'concrete';
 
+    if (field?.runtimeField?.parent !== undefined) {
```

Comment:
For the demo we prevent editing a subField directly as this will have to be done by editing the runtime object script.